management/src/main/java/com/bitwisekaizen/sdss/dto/IscsiTargetSpec.java

```diff
@@ -0,0 +1,49 @@
+package com.bitwisekaizen.sdss.dto;
+
+import java.util.List;
+
+/**
+ * Spec that describes the ISCSI target
+ */
+public class IscsiTargetSpec {
+
+    private List<String> hostIscsiQualifiedNames;
+    private int capacityInMb;
+    private String targetName;
+
+    // Json serialization
+    private IscsiTargetSpec(){}
+
+    public IscsiTargetSpec(List<String> hostIscsiQualifiedNames, int capacityInMb, String targetName) {
```

Comment:
Might want to shoot for an abstraction with naming similar to the following:

IscsiTarget(IscsiAccessControlList acl, int cap, String deviceName)

IscsiTarget(IscsiACL acl, IscsiDevice device)

IscsiDevice(int cap, String deviceName)

IscsiXXXX(IscsiTarget target, String storageIp)
+ uuid:UUID

UniquelyIdentifiableIscsiTarget - target that's unique within the entire storage service.